src/components/buildArguments.ts

```diff
@@ -20,17 +20,10 @@ export const buildArguments = (
 		args.push('-o', singleQuotify(storageUri.fsPath));
 		args.push('-l', command.language);
 
-		// configuration.includePatterns.forEach((includePattern) => {
-		// 	const { fsPath } = Uri.joinPath(message.targetUri, includePattern);
-
-		// 	args.push('-p', singleQuotify(fsPath));
-		// });
-
-		// configuration.excludePatterns.forEach((excludePattern) => {
-		// 	const { fsPath } = Uri.joinPath(message.targetUri, excludePattern);
-
-		// 	args.push('-a', singleQuotify(fsPath));
-		// });
+		const codemodArguments = (command.arguments ?? []).flatMap(
```

Comment:
if it's used in two places, I would calculate it before the split by `command.kind`